examples/webgl_loader_texture_rgbm.html

```diff
@@ -44,14 +46,10 @@
 
 				camera = new THREE.OrthographicCamera( - aspect, aspect, 1, - 1, 0, 1 );
 
-				new THREE.TextureLoader().load( 'textures/memorial.png', function ( texture ) {
+				new RGBMLoader().load( 'textures/memorial.png', function ( texture ) {
 
 					texture.encoding = THREE.RGBM16Encoding;
```

Comment:
Shouldn't `RGBMLoader` set this encoding by default?

Reply:
There is also `RGBM7Encoding`. I don't think it's possible to detect which encoding is used so I thought it's better to set it in the example code.

Reply:
Hmm, do we need `RGBM7Encoding`? 😅

Considering this was not working on iOS, I suspect this doesn't have't that much real usage...

@vjevremovic do we need both `RGBM7` and `RGBM16`? Or can we just pick one?

```js
export const RGBEEncoding = 3002;
export const LogLuvEncoding = 3003;
export const RGBM7Encoding = 3004;
export const RGBM16Encoding = 3005;
export const RGBDEncoding = 3006;
```

I'm not sure we need that many encodings...

/cc @bhouston

Reply:
I'd default the encoding to `RGBM7Encoding`. At least it will render correctly up to an exposure factor. Defaulting the encoding to `Linear` is just wrong.

Reply:
Although technically not correct, I've read that the resulting artifacts due to linear filtering with RGBM or RGBD are considered as acceptable. This explains why you see the usage of linear filtering with both approaches. 

But I understand that it depends on the texture data and the personal preference what kind of filtering is appropriate. 

@elalish What is your recommendation? Should we keep the usage of linear filtering? Or default to nearest?

Reply:
@Mugen87 Another option is to add 2 classes (`RGBM7Loader` and `RGBM16Loader`) in RGBMLoader.js that extend `RGBMLoader`.

Reply:
If we decide to go this route I would like to see an `EquirectangularTextureLoader` (which is derived from `TextureLoader`) so users don't have to manually set `textureEquirec.mapping = THREE.EquirectangularReflectionMapping;` when they load such a texture for the background.

Reply:
In this case, we should reconsider the concept of `EquirectangularTextureLoader` and `EquirectangularTexture`, see https://github.com/mrdoob/three.js/pull/19845#issuecomment-661733154.

Reply:
I made up RGMB16 as the original RGBM code used a constant of 5, 6, 7 or 8: http://graphicrants.blogspot.com/2009/04/rgbm-color-encoding.html. I found RGBM16 a decent balance of precision and range.  I think it is fair to get rid of RGBM7.

Reply:
Hmm, I'll merge and experiment locally.

Reply:
So how do users other than @bhouston create RGBM files? Do they use a Blender add-on or something?

Reply:
@Mugen87 I, too, think Linear encoding is acceptable, even if not correct. I am not sure about mip generation, though. I guess defaulting to false is OK.

Reply:
I would like to know that too...

Reply:
Three.js has the ability to take a RGBE/HDR texture and encode it in RGBM just rendering from one texture to another using an encoding.

To make it easier one could make a Three.js example that could transcode between all these image formats pretty quickly.  webgl_texture_encodings.html maybe?  I seem to remember this suggested like 4 years ago.

Reply:
I've hacked something together to test the RGBE to RGBM conversion for `webgl_texture_encodings`. The following fiddle loads a RGBE texture, converts it to RGBM and outputs the PNG on the website.

https://jsfiddle.net/ak6en57b/2/

However, I've encountered an issue. It seems the encoding of the resulting RGBM texture is not correct. If I use the result in the RBGM example, it looks wrong, see https://jsfiddle.net/jg2so8mn/

I wonder what bit is missing 🤔 ...

Reply:
Never mind! I've resolved the issue my manually encoding the PNG via `UPNG.js` and not using canvas. Updated fiddle:

https://jsfiddle.net/2do9arLf/1/

Not sure but it seems when applying the image data to the canvas, a color space conversion was applied on the texels.